Getting-Started.md

```diff
@@ -0,0 +1,6 @@
+# Getting started with Draco
+First you need to have [vscode](https://code.visualstudio.com/download) and [dotnet](https://dotnet.microsoft.com/en-us/download) installed.  
+In vscode install `Draco language support` extension.
```

Comment:
```suggestion
In VS Code install `Draco language support` extension.
```

Reply:
You can also just link the extension from the marketplace: https://marketplace.visualstudio.com/items?itemName=Draco-lang.draco-language-support

Reply:
The user might not want to install VS Code, heck it's not even a requirement.